pkg/pdfcpu/model/equal.go

```diff
@@ -141,7 +142,7 @@ func equalFontNames(v1, v2 types.Object, xRefTable *XRefTable) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	bf2 := v2.(types.Name)
+	bf2, ok := v2.(types.Name)
 	if !ok {
 		return false, errors.Errorf("equalFontNames: type cast problem")
```

Comment:
If this is added, it fixes parsing for another pdf...
name, ok := o.(types.Name)
if !ok {
nameInt, ok := o.(types.Integer)
if !ok {
return nil, errors.Errorf("pdfcpu: validateNameEntry: dict=%s entry=%s invalid type %T", dictName, entryName, o)
}
name = types.Name(nameInt.String())
}